README.md

```diff
@@ -0,0 +1,98 @@
+# OpenSergo Go SDK
+
+
+## How to use
+
+### scene 1 : subscribe config-data
+``` go
+package main
+
+func main() {
+    // add console logger (optional)
+    //logging.NewConsoleLogger(logging.InfoLevel, logging.SeparateFormat)
+    // add file logger (optional)
+    //logging.NewFileLogger("/logs/opensergo/opensergo-universal-transport-service.log", logging.InfoLevel, logging.JsonFormat)
+    
+    // instant OpenSergoClient
+    openSergoClient := client.NewOpenSergoClient("33.1.33.1",10246)
+    
+    // registry SubscribeInfo of FaultToleranceRule
+    // 1. instant SubscribeKey
+    faultToleranceSubscribeKey := subscribe.NewSubscribeKey("default", "foo-app", configkind.ConfigKindRefFaultToleranceRule{})
+    // 2. construct SubscribeInfo
+    faultToleranceSubscribeInfo := client.NewSubscribeInfo(faultToleranceSubscribeKey)
+    // 3. registry
+    openSergoClient.RegisterSubscribeInfo(faultToleranceSubscribeInfo)
+    
+    // start OpensergoClient
+    openSergoClient.Start()
+    
+    // registry after OpenSergoClient started
+    // registry SubscribeInfo of RateLimitStrategy
+    rateLimitSubscribeKey := subscribe.NewSubscribeKey("default", "foo-app", configkind.ConfigKindRefRateLimitStrategy{})
+    rateLimitSubscribeInfo := client.NewSubscribeInfo(rateLimitSubscribeKey)
+    openSergoClient.RegisterSubscribeInfo(rateLimitSubscribeInfo)
+
+    select {}
+}
+```
+
+### scene 2 : subscribe config-data with custom-logic when config-data changed.
+Add a subscriber by implements the function in `subscribe.Subscriber`.  
+There are some samples in `sample` directory : `sample/main/sample_faulttolerance_rule_subscriber.go` and `sample_ratelimit_strategy_subscriber.go`
+``` go
+    type SampleFaultToleranceRuleSubscriber struct {
+    }
+    
+    func (sampleFaultToleranceRuleSubscriber SampleFaultToleranceRuleSubscriber) OnSubscribeDataUpdate(subscribeKey subscribe.SubscribeKey, dataSlice []protoreflect.ProtoMessage) bool {
+        // TODO add custom-logic when config-data change
+        // ......
+        return true
+    }
+```
+And then registry it to `subscriber.SubscriberRegistry`.  
+Demo-code is in `sample/main` directory
```

Comment:
## Demo

For more info, please refer to [samples](/samples)

Reply:
optimized.